modules/monitoring-optimizing-alerting-rules-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: CONCEPT
+[id="Optimizing-alerting-rules-for-core-platform-monitoring{context}"]
+= Optimizing alerting rules for core platform monitoring
+
+You can optimize alerting for core platform monitoring by considering the following recommendations when creating or modifying alerting rules:
+
+* *Minimize the number of new alerting rules that you create*.
+Create only those new alerting rules that are essential to your specific requirements.
+It can be difficult for users to notice important alerts if the system is generating numerous, less critical alerts for conditions that might not be relevant to a given situation.
```

Comment:
`It can be difficult for users to notice important alerts if the system is generating numerous`
->
`It can be difficult for users to notice important alerts if the system is generating numerous alerts`

`less critical alerts for conditions that might not be relevant to a given situation`
it seems strange, but I don't get the point

Reply:
It's a bit confusing, you're right. Removing.